.circleci/config.yml

```diff
@@ -217,20 +232,44 @@ jobs:
     working_directory: /go/src/github.com/kubeapps/kubeapps
     environment:
       GOPATH: /home/circleci/.go_workspace
-      IMAGES: "kubeapps/apprepository-controller kubeapps/tiller-proxy"
+      IMAGES: "apprepository-controller tiller-proxy"
     <<: *build_images
   build_dashboard:
     docker:
       - image: circleci/golang:1.11
     environment:
-      IMAGES: "kubeapps/dashboard"
+      IMAGES: "dashboard"
     <<: *build_images
   release:
     docker:
       - image: circleci/golang:1.11
     steps:
       - checkout
       - run: REPO_DOMAIN=kubeapps REPO_NAME=kubeapps ./script/create_release.sh ${CIRCLE_TAG}
+  local_e2e_tests:
+    working_directory: /home/circleci/.go_workspace/src/github.com/kubeapps/kubeapps
+    machine: true
+    environment:
+      GOPATH: /home/circleci/.go_workspace
+    steps:
+      - checkout
+      - <<: *exports
+      # By default Go 1.9 is installed, we need 1.11 to use kind
+      - run: |
+          sudo rm -rf /usr/local/go/*
+          sudo chown $(whoami) /usr/local/go
+          curl -LO https://dl.google.com/go/go1.11.9.linux-amd64.tar.gz
+          tar -C /usr/local -xzf go1.11.9.linux-amd64.tar.gz
+      - run: go get -u sigs.k8s.io/kind
```

Comment:
Can we use a pre-built binary from a release instead? https://github.com/kubernetes-sigs/kind/releases. This should avoid the need to install Go 1.11?

Reply:
the only release available is a prerelease. I am not sure if it will work but I can try, probably it's enough.